README.md

```diff
@@ -57,6 +57,11 @@ curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -
 ```
 curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b <DESTINATION_DIR> <RELEASE_VERSION>
 ```
+### Windows
```

Comment:
Hey @marcinbojko! Thanks for the PR. Would you be able to add some additional details to this windows block.

"The chocolatey distribution of syft is community maintained and not distributed by the anchore team"

Also update "Windows" => "Chocolatey" to keep it consistent with how we label "Homebrew". Other Windows users might want other distribution options so we don't want to only point them to chocolatey.

Thanks again!

Reply:
Hi @spiffcs 
 Changes applied. The same BTW will happen with 'grype' ;)